src/view/components/grid/grid.js

```diff
@@ -66,16 +69,29 @@ export class Grid extends RootComponent {
 		});
 	}
 
-	applyFactory(gf) {
+	applyFactory(gf = null, mode = 'async') {
```

Comment:
it;s like of globalF and localF, however i'm still thinkinkg how to get rid of that applyFactory